bokehjs/src/coffee/models/renderers/renderer.coffee

```diff
@@ -286,19 +286,6 @@ class RendererView extends BokehView
 class Renderer extends Model
   type: "Renderer"
 
-  _coords: []
-
-  @coords: (coords) ->
-    _coords = this.prototype._coords.concat(coords)
-    this.prototype._coords = _coords
-
-    result = {}
-    for [x, y] in coords
-      result[x] = [ p.NumberSpec ]
-      result[y] = [ p.NumberSpec ]
-
-    @define(result)
-
```

Comment:
@mattpap I moved this to be able to bring the class hierarchy clkoser into accord. `Glyph` shloud inherit from `Model` (not `Renderer`) and this change allows that to happen on the JS side.